.github/workflows/qa-azure.yaml

```diff
@@ -23,6 +23,12 @@ jobs:
     steps:
       - name: Check out repository
         uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: go.work
+      - name: Set up version
```

Comment:
We are not yet ingesting this information into the binaries, are we?

Reply:
No, not yet.